ReactiveXComponent/Connection/IXCSubscriber.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using ReactiveXComponent.RabbitMq;
+using ReactiveXComponent.RabbitMQ;
+
+namespace ReactiveXComponent.Connection
+{
+    public interface IXCSubscriber : IDisposable
+    {
+        void AddCallback(string component, string stateMachine, Action<MessageEventArgs> callback);
```

Comment:
perhaps subscribe and unsubscribe are better naming for this interface.
callback is too generic. stateMachineUpdateListener or stateMachineUpdateCallback are better...